
project bionic/
diff --git a/bionic/libc/Android.mk b/bionic/libc/Android.mk
index e8775ae..31a79a2 100644
--- a/bionic/libc/Android.mk
+++ b/bionic/libc/Android.mk
@@ -628,6 +628,11 @@ ifeq ($(TARGET_ARCH),$(filter $(TARGET_ARCH),mips mips64))
   use_clang := false
 endif
 
+# Disable clang for targets who specify it
+ifeq ($(TARGET_BIONIC_DISABLE_CLANG),true)
+  use_clang := false
+endif
+
 ifeq ($(use_clang),)
   use_clang := true
 endif
